src/components/Alert/Alert.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { select, text as textKnob } from '@storybook/addon-knobs';
+import { Alert } from './Alert';
+
+import {
+  Variants,
+  Types,
+} from './Alert.styles';
+
+export default {
+  component: Alert,
+  parameters: {
+    componentSubtitle: 'An alert displays a short, important message in a way that attracts the user\'s attention without interrupting the user\'s task.',
+  },
+  title: 'Components|Alert',
+};
+
+const types = {
+  error: 'error',
+  info: 'info',
+  success: 'success',
+  warning: 'warning',
+};
+
+const variants = {
+  standard: 'standard',
+};
+
+const message: string = 'Lorem ipsum dolor sit amet consectetur adipisicing elit.';
```

Comment:
Eu recomendo usar uma frase placeholder em inglês, pra evitar esse falso alarme do typocheck. Pode ser a descrição **"An alert displays a short, important message in a way that attracts the user\'s attention without interrupting the user's task."**, que você colocou na linha 13 mesmo.